test/e2e/utils/chaise.page.js

```diff
@@ -775,6 +778,10 @@ var recordPage = function() {
     this.getLoadingElement = function () {
         return element(by.id("rt-loading"));
     }
+
+    this.getElementById = function (id) {
```

Comment:
This selector is unnecessary. It doesn't add any value to call `chaisePage.recordPage.getElementById(id)` instead of `element(by.id(id))` (which is much shorter). Our `chaisePage` functions are meant to be reusable functions that we use multiple times to fetch specific elements on the page.